apps/console/src/extensions/configs/models/user.ts

```diff
@@ -27,4 +27,5 @@ export interface User {
     enableBulkImportSecondaryUserStore: boolean;
     deleteUser: (user: ProfileInfoInterface) => Promise<any>;
     disableManagedByColumn: boolean;
+    enableAdminPrivilegeRevokeOption: boolean;
```

Comment:
Let's send the cloud config PR as well, along with this PR.

Reply:
Will send the PR with the version bump.